spec/src/main/asciidoc/changelog.adoc

```diff
@@ -30,6 +30,10 @@ Changes marked with icon:bolt[role="red"] are breaking changes relative to previ
 ** Added the `MetricID.getTagsAsArray()` method to the API.
 ** Added the method `MetricType.fromClassName`
 ** Improved TCK - Use WebArchive for deployment
+** Introduce two new optional metrics derived from RESTful stats into the base scope.
```

Comment:
Should be higher in the list as this is kind of a selling point. I'd place it 2nd right after SimpleTimer.